src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php

```diff
@@ -57,6 +57,14 @@ public function iExpectAConfirmationDialogShouldAppear()
     }
 
     /**
+     * @Given I click the back icon
+     */
+    public function iClickOnTheBackIcon()
+    {
+        $this->clickSelector('.fa-chevron-left');
```

Comment:
This is currently not working and therefore not being used, right? Then I would remove it for now (since it might have to be solved in a different way anyway)